examples/snake/main.go

```diff
@@ -102,6 +102,10 @@ func (g *Game) reset() {
 }
 
 func (g *Game) Update() error {
+	//Here, in order to prevent it from suddenly going in the opposite direction in the movement logic,
+	//we block the value used to move to the opposite side with an if statement,
```

Comment:
If possible, avoid using `we` (since it is not clear who 'we' are)

Reply:
Im sorry I directly copy and pasted from translate I hate translate. wait I will fix it